MekHQ/src/mekhq/campaign/finances/Money.java

```diff
@@ -154,6 +154,7 @@ public String toAmountAndSymbolString() {
         return CurrencyManager.getInstance().getUiAmountAndSymbolPrinter().print(getWrapped().toMoney(RoundingMode.HALF_EVEN));
     }
 
+    @Deprecated
```

Comment:
I deprecated rather than removing to not cause issues with any potential in-flight PRs.